static/src/stylesheets/module/commercial/_adverts-soulmates.scss

```diff
@@ -96,6 +108,7 @@
 
 .advert-blended--soulmates {
     background: $soulmates-background;
+    box-sizing: border-box;
```

Comment:
In this case the combination of `padding` and `flex-basis` drives Chrome crazy and prevents it from breaking words; `border-box` solves this.